### PR TITLE
Fix the aggregate result problem in UfsIOBench

### PR DESCRIPTION
**Pull request by bzheng888**
### What changes are proposed in this pull request?

Fix the aggregate result problem in UfsIOBench.

### Why are the changes needed?
The  UfsIOBench result summary calculates the total duration by summing each  points' result, and   compute the average speed based on total duration, actually the average can't  represent the system performance. 
For examples: 
An alluxio cluster with 1 job master and 3 job worker,  Run the UfsIOBench with 16 threads:
```
./bin/alluxio runUfsIOTest --path hdfs://hdfsCluster/tmp/ufsIoBanch --io-size 2G--threads 16 --cluster
```
WeChatWorkScreenshot_14f71ae8-abe3-410f-bc45-5d31e45e1691

the mTotalDurationSeconds is 16 * 3 *(points duration), but actually the 16 points read and write concurrently  and finished almostly at same time.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
No

### CODE REVIEW

**Review by dbw9580 — CHANGES_REQUESTED** [review]

**Comment by bzheng888** [comment]
@dbw9580 PTAL, thx!

**Comment by bzheng888** [comment]
@dbw9580 Any suggestion on this pr?

File: stress/common/src/main/java/alluxio/stress/worker/IOTaskSummary.java [hunk 1]
```diff
@@ -151,7 +152,10 @@ public static class SpeedStat implements JsonSerializable {
     public long mTotalSizeBytes;
     public double mMaxSpeedMbps;
     public double mMinSpeedMbps;
+    // Average speed of all points
     public double mAvgSpeedMbps;
+    // The average speed to write/read UFS on the cluster view
+    public double mClusterAvgSpeedMbps;
     public double mStdDev;
```
Comment (dbw9580) [inline stress/common/src/main/java/alluxio/stress/worker/IOTaskSummary.java:159]:
```suggestion
    // Max speed among all nodes
    public double mMaxSpeedMbps;
    // Min speed among all nodes
    public double mMinSpeedMbps;
    // Average speed of all nodes
    public double mAvgSpeedMbps;
    // Cluster-wide throughput
    public double mClusterAvgSpeedMbps;
    // Standard deviation of speed reported by each node
    public double mStdDev;
```

**Review by dbw9580 — APPROVED** [review]
LGTM

**Comment by bzheng888** [comment]
@jiacheliu3 PTAL!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**